docs/user-guide/deployments.md

```diff
@@ -453,6 +453,55 @@ nginx-deployment-3066724191   0         0         1h
 
 Note: You cannot rollback a paused Deployment until you resume it.
 
+## Proportional scaling
+
+RollingUpdate Deployments support running multitple versions of an application at the same time. When there
+is a new rollout ongoing or it is paused (you have multiple running versions in both cases) and a scaling
```

Comment:
Suggest make it shorter and easier to read, something like "When you scale a Deployment that is either still in progress or paused, ..."